main.py

```diff
@@ -1,10 +1,77 @@
+import json
+from typing import List
+
 import init_django_orm  # noqa: F401
 
 from db.models import Race, Skill, Player, Guild
 
 
+def receive_player_race(race: dict[str]) -> None:
```

Comment:
Kinda weird naming with "receive" in the beginning, I'd say better name it "create" because you are not receiving anything from those functions.